pages/support.md

```diff
@@ -0,0 +1,75 @@
+<script>
+{
+	"title": "jQuery Support"
+}
+</script>
+
+## Community support
+
+The following learning resources are available online and free of charge:
+
+* [jQuery Learning Center](https://learn.jquery.com/)
+* [jQuery API Documentation](https://api.jquery.com/)
+* [jQuery UI Demos](https://jqueryui.com/demos/)
+* [jQuery UI API Documentation](https://api.jqueryui.com/)
+
+### Matrix chat
+
+jQuery is on Matrix! The support channel for jQuery is [#jquery_meeting:gitter.im](https://app.element.io/#/room/#jquery_meeting:gitter.im). You can [read the channel on Element](https://app.element.io/#/room/#jquery_meeting:gitter.im) without an account, or [join the channel](https://matrix.to/#/#jquery_meeting:gitter.im) via any Matrix client.
+
+We use the public [#jquery_dev:gitter.im channel](https://app.element.io/#/room/#jquery_dev:gitter.im) to discuss project developments, and also hold [weekly meetings](https://meetings.jquery.org/).
+
+### IRC chat
+
+The IRC support channel for jQuery projects is `#jquery` on [Libera Chat IRC](https://libera.chat/). You can [join via webchat](https://web.libera.chat/#jquery), or via an IRC client.
+
+### StackOverflow
+
+StackOverflow is a great place to ask questions and find support for all jQuery projects. Search or create [questions tagged with "jquery"](https://stackoverflow.com/questions/tagged/jquery), or refer to the ["jquery" tag introduction](https://stackoverflow.com/tags/jquery/info).
+
+### Archives
+
+* [Freenode IRC chat channels](https://irc.jquery.org/) (2011-2021)
+* [jQuery Forum](https://forum.jquery.com/) (2010-2021): For questions and advice regarding jQuery Core, jQuery UI, Themeroller, QUnit, development of jQuery Plugins, and more.
+* [jQuery Accessibility mailing list](https://groups.google.com/group/jquery-a11y) (2008-2016).
+
+## Commercial support
+
+The following member companies offer commercial support services for jQuery, including consulting and training.
```

Comment:
Is this still true?

Reply:
The phrasing used on older pages was that they are sponsors. I removed that part, but figured it might be good for the project to keep listing the ability to have commercial support. And, if we remain comfortable/friendly towards these, we could keep mentioning them.

But yeah, I don't know if they actually still are willing to do consulting in this area. My gut says yes but worth checking!